bower.json

```diff
@@ -9,23 +9,28 @@
     "polymer",
     "progress"
   ],
-  "main": "paper-progress.html",
+  "main": "paper-linear-progress.html",
   "repository": {
     "type": "git",
-    "url": "git://github.com/PolymerElements/paper-progress.git"
+    "url": "git://github.com/PolymerElements/paper-linear-progress.git"
   },
   "dependencies": {
     "polymer": "Polymer/polymer#^1.1.0",
-    "iron-range-behavior": "PolymerElements/iron-range-behavior#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "iron-flex-layout": "PolymerElements/iron-flex-layout#^1.0.0"
+    "neon-animation": "PolymerElements/neon-animation#^1.0.0"
   },
   "devDependencies": {
     "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
     "test-fixture": "PolymerElements/test-fixture#^1.0.0",
-    "paper-button": "PolymerElements/paper-button#^1.0.0",
     "web-component-tester": "*",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0"
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
+    "paper-checkbox": "PolymerElements/paper-checkbox#^1.0.0",
+    "paper-input": "PolymerElements/paper-input#^1.0.0",
+    "iron-flex-layout": "PolymerElements/iron-flex-layout#^1.0.0",
+    "es6-promise": "~3.0.2"
```

Comment:
We have a preferred Promise polyfill for usage in tests and demos: `PolymerLabs/promise-polyfill`.